framework/include/base/FEProblemBase.h

```diff
@@ -665,6 +666,13 @@ class FEProblemBase :
   MooseSharedPointer<MultiApp> getMultiApp(const std::string & multi_app_name);
 
   /**
+   * Execute MultiAppTransfers associate with execution flag and direction.
+   * @param type The execution flag to execute.
+   * @param direction The direction (to or from) to transfer.
+   */
+  void execMultiAppTransfers(ExecFlagType type, MultiAppTransfer::DIRECTION direction);
```

Comment:
Looks like you may have deleted the old method? Did you forget to delete it from the header? This new signature requires a hew header file to be added as you did so I liked the old signature better. Just a comment.